setup.py

```diff
@@ -270,6 +270,7 @@ def long_description():
 
     # Additional scripts.
     scripts=[
+        'examples/pylink-rtt'
```

Comment:
Do you know if this works on Windows, or would we have to do `os.path.join()`?

Reply:
I don't actually know but I changed it. This was an oversight on my part, I think it's a good idea to always use `os.path.join()`.